apps/site/lib/site_web/controllers/schedule/schedule_api.ex

```diff
@@ -56,6 +56,7 @@ defmodule SiteWeb.ScheduleController.ScheduleApi do
 
   def enhance_services([]), do: []
 
+
```

Comment:
How about rebasing to remove this commit entirely, since now it's not doing anything but adding some whitespace?

Reply:
...and also the next commit.

Reply:
I always squash and merge so those commits go away...

It's not a habit on our team to rebase out commits in a topic branch. I can do it now though.

Reply:
If you see in the final diff, there's no extra space, this commit was pre-format (there's a commit called "format")